corehq/apps/domain/templates/domain/admin/partials/bootstrap5/project_limits_table.html

```diff
@@ -23,8 +23,8 @@ <h1>{{ header }}</h1>
           <td>{{ rate_info.key }}</td>
           <td>
             {{ rate_info.percent_usage }}%
-            <div class="progress">  {# todo B5: css-progress #}
-              <div class="progress-bar progress-bar-striped" style="width: {{ rate_info.percent_usage }}%"></div>  {# todo B5: inline-style #}
+            <div class="progress" role="progressbar" aria-label="Percent Usage" aria-valuemin="0" aria-valuemax="100">
```

Comment:
Should "Percent Usage" be translated?

Reply:
[The comment ](https://github.com/dimagi/commcare-hq/blob/9c6bbc2e6e1e76100f2fd73580fb06249e4fc40b/corehq/apps/domain/templates/domain/admin/partials/bootstrap5/project_limits_table.html#L2-L4)in the template explained it is just for internal use so it won't be translated